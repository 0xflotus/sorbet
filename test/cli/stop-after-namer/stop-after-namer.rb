# typed: true

class A
  def method(method_arg)
    local = 1
    @field = 2
    $global = 3
  end

  def self.singleton_method(singleton_method_arg)
    @singleton_field = 4
  end

  @@static_field = 5
end
