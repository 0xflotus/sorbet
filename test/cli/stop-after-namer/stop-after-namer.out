No errors! Great job.
class ::<root> ()
  field #$global @ test/cli/stop-after-namer/stop-after-namer.rb:7
  class ::A < ::<todo sym> () @ test/cli/stop-after-namer/stop-after-namer.rb:3
    method ::A#method (method_arg) @ test/cli/stop-after-namer/stop-after-namer.rb:4
      argument ::A#method#method_arg<> @ test/cli/stop-after-namer/stop-after-namer.rb:4
  class ::<Class:A> < ::<todo sym> () @ test/cli/stop-after-namer/stop-after-namer.rb:3
    method ::<Class:A>#singleton_method (singleton_method_arg) @ test/cli/stop-after-namer/stop-after-namer.rb:10
      argument ::<Class:A>#singleton_method#singleton_method_arg<> @ test/cli/stop-after-namer/stop-after-namer.rb:10

